plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -515,6 +518,17 @@ private String createContainer(String workspaceId,
                                            .collect(toMap(Function.identity(),
                                                           value -> new PortBinding[0])))
                   .withVolumesFrom(toArrayIfNotNull(service.getVolumesFrom()));
+        
+        if (volumesFrom != null) {
```

Comment:
I've added a new unit test for the volumesFrom parameter, some of the API in the test framework was a little confusing but hopefully it's ok.